modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -20,7 +20,7 @@ class ClaimsAndAppealsController < ApplicationController
       end
 
       def index
-        validated_params = validate_params(params)
```

Comment:
In addition to the pagination changes, I changed some of the params handling, because params is already defined in the controller and I found passing them around under the arg name "params" confusing.